apps/src/tutorialExplorer/roboticsButton.jsx

```diff
@@ -28,14 +45,23 @@ const RoboticsButton = React.createClass({
     };
 
     return (
-      <div style={{float:"left"}}>
+      <div>
         <div style={{display: getResponsiveValue({md: "block", xs: "none"})}}>
-          <a href="/learn/robotics">
-            <img src="/images/learn/robotics-link.png" style={styles.roboticsButtonImage}/>
-          </a>
+          <div style={styles.button}>
+            <a href="/learn/robotics">
+              <div style={{position: "relative"}}>
+                <img src="/images/learn/robotics-link.png" style={styles.roboticsButtonImage}/>
+                <div style={styles.roboticsButtonText}>
+                  {i18n.roboticsButtonText()}
+                  &nbsp;
+                  <i className="fa fa-arrow-right" aria-hidden={true}/>
+                </div>
+              </div>
+            </a>
+          </div>
         </div>
         <div style={roboticsTextStyle}>
-          {i18n.roboticsButtonText({robotics_url: "/learn/robotics"})}
+          <div dangerouslySetInnerHTML={{__html: i18n.roboticsText({robotics_url: "/learn/robotics"})}}/>
```

Comment:
@Bjvanminnen you were absolutely right that this was needed.. I'm bewildered as to why it seemed to work before when I explicitly checked.